setup.py

```diff
@@ -10,7 +10,7 @@
     author="Department for Levelling Up, Housing and Communities",
     description="Provides access to functionality common to creating a data dashboard.",
     name="gov_uk_dashboards",
-    version="9.43.0",
+    version="9.43.1",
```

Comment:
On reflection, this might actually, be a major. What do you think?
```suggestion
    version="10.0.0",
```

Reply:
I think major feels better!